utils/__tests__/config.test.ts

```diff
@@ -0,0 +1,199 @@
+import { Either } from "fp-ts/lib/Either";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { MailerConfig } from "../config";
+
+const aMailFrom = "example@test.com";
+
+const noop = () => {};
+const expectRight = <L, R>(e: Either<L, R>, t: (r: R) => void = noop) =>
+  e.fold(
+    _ =>
+      fail(`Expecting right, received left. Value: ${JSON.stringify(e.value)}`),
+    _ => t(_)
+  );
+
+const expectLeft = <L, R>(e: Either<L, R>, t: (l: L) => void = noop) =>
+  e.fold(
+    _ => t(_),
+    _ =>
+      fail(`Expecting left, received right. Value: ${JSON.stringify(e.value)}`)
+  );
+
+describe("MailerConfig", () => {
+  it("should decode configuration for sendgrid", () => {
+    const rawConf = {
+      MAIL_FROM: aMailFrom,
+      NODE_ENV: "production",
+      SENDGRID_API_KEY: "a-sg-key"
+    };
+    const result = MailerConfig.decode(rawConf);
+
+    expectRight(result, value => {
+      expect(value.SENDGRID_API_KEY).toBe("a-sg-key");
+      expect(typeof value.MAILUP_USERNAME).toBe("undefined");
+    });
+  });
+
+  it("should decode configuration for sendgrid", () => {
+    const rawConf = {
+      MAIL_FROM: aMailFrom,
+      NODE_ENV: "production",
+      SENDGRID_API_KEY: "a-sg-key"
+    };
+    const result = MailerConfig.decode(rawConf);
+
+    expectRight(result, value => {
+      expect(value.SENDGRID_API_KEY).toBe("a-sg-key");
+      expect(typeof value.MAILUP_USERNAME).toBe("undefined");
+    });
+  });
```

Comment:
This is a duplicate test

Reply:
Do yo mean the whole case or just `expect(typeof value.MAILUP_USERNAME).toBe("undefined");`?

Reply:
The whole test is exactly the same as the previous one

Reply:
oops, didn't notice that. Good catch!